src/main/flight/mixer.c

```diff
@@ -62,9 +63,11 @@
 #include "flight/mixer.h"
 #include "flight/mixer_tricopter.h"
 #include "flight/pid.h"
+#include "flight/position.h"
 
 #include "rx/rx.h"
 
+#include "sensors/barometer.h"
```

Comment:
just a few have barometer.

Reply:
ok got it ! thanks @gretel i will improve this